test/LifToken.js

```diff
@@ -32,6 +32,24 @@ contract('LifToken', function(accounts) {
     await help.checkValues(token, accounts,1000000, 10000000, 0, [4000000,3000000,2000000,1000000,0]);
   });
 
+  it("should be able to stop and resume the token", async function() {
+    await token.stop();
+
+    await help.addCrowdsaleStage(token, 10000000, price)[1];
+
+    var price = web3.toWei(0.1, 'ether');
+
+    assert.equal(await token.submitBid(accounts[1], 1000, { value: 1000*price, from: accounts[1]}),
+      false, // TODO: this doesn't work of course, how can I assert that the tx failed?
```

Comment:
@nberger adding another uint and function on the token contract is not necessary, you can use setStatus() function.
The setStatus() function can be called by the owner or DAO, the idea is to test it on both ways.
https://github.com/windingtree/LifToken/blob/master/contracts/LifToken.sol#L409

https://github.com/windingtree/LifToken/blob/master/test/DAO.js#L175 You can use this test to test the setStatus form DAO proposal